ament_cmake_gen_version_h/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>ament_cmake_gen_version_h</name>
+  <version>1.1.4</version>
+  <description>Generate a C header containing the version number of the package</description>
+  <maintainer email="serge@safeai.ai">Serge Nikulin</maintainer>
+  <license>Apache License 2.0</license>
+
+  <author>Serge Nikulin</author>
+
+  <buildtool_depend>ament_cmake_core</buildtool_depend>
+  <buildtool_depend>ament_package</buildtool_depend>
+
+  <buildtool_export_depend>ament_cmake_core</buildtool_export_depend>
+  <test_depend>ament_cmake_gtest</test_depend>
```

Comment:
@serge-nikulin consider adding `ament_lint_common` linters.

Reply:
@hidmic, `ament_lint` repo depends on `amen_cmake` and does not exist during `amen_cmake` build. Nevertheless, on my PC I did run all the linters "out of order" and fixed all violations.

Reply:
Ah good point. 

> I did run all the linters "out of order" and fixed all violations.

Much appreciated.